README.md

```diff
@@ -75,6 +75,7 @@ Here are the default settings:
   cwd_change_handling = false, -- Follow cwd changes, saving a session before change and restoring after
   lsp_stop_on_restore = false, -- Should language servers be stopped when restoring a session. Can also be a function that will be called if set. Not called on autorestore from startup
   restore_error_handler = nil, -- Called when there's an error restoring. By default, it ignores fold errors otherwise it displays the error and returns false to disable auto_save
+  purge_after_days = nil, -- Sessions older than purge_after_days will be deleted asynchronously on startup, defaults to off (no purging), requires >= nvim 0.10
```

Comment:
I wonder if days is really the best granularity we can get? I can see people wanting to set hours or minutes for example (probably not less than that but who knows). Ultimately I don't think there's a point in limiting ourselves to days really

Reply:
yeah, i went back and forth on that, trying to figure out how to balance of simplicity and expressiveness. specifying it as seconds is definitely more expressive but i thought that was more complicated for maybe not enough benefit. i can change it, tho.

how about:

```lua
purge_after_minutes = nil -- Sessions older than purge_after_minutes will be deleted asynchronously on startup, e.g. set to 14400 to delete sessions that haven't been accessed for more than 10 days, defaults to off (no purging), requires >= nvim 0.10
```

Reply:
Yeah I think we can go with minutes then. Thanks 👍